### DISCUSSION TITLE
can't get collection's document by 'get_doc()' function

### DISCUSSION THREAD

**Discussion by zlzonchan** [GodotNuts/GodotFirebase#444] [Q&A] [answered]
`var collection : FirestoreCollection = Firebase.Firestore.collection('user stages')
var document : FirestoreDocument = await collection.get_doc('sans')`

I want to get document named "sans" in "user stages" collection. But I got this error message.
`[Firebase Error] >> Action in error was: 0 (GET DOCUMENT)
get_document returned null for user stages sans`

how to fix this error? I searched for hours but couldn't find solution. 
(please understand I'm foreigner and not good at English.)

**Comment by WolfgangSenff** [comment, marked as answer]
Generally I'd expect that to work. Try removing the space from the collection name in Firebase and in your game. I've noticed an uptick in people complaining when there are weird characters in the names of collections or documents causing issues.

**Reply by zlzonchan** [reply]
You're right and now I fix it! thank you very much!
